controllers/rhmi/rhmi_controller.go

```diff
@@ -271,6 +272,18 @@ func (r *RHMIReconciler) Reconcile(ctx context.Context, request ctrl.Request) (c
 		return ctrl.Result{}, err
 	}
 
+	alertsClient, err := k8sclient.New(r.mgr.GetConfig(), k8sclient.Options{
```

Comment:
moving the creation of the alerts as early as possible in the reconciler

Reply:
Just out of interest, what's the exact benefit of moving it as early as possible? 
If RHOAM fails due to pre-flight checks, do we want the alerts to fire, if so, maybe it's worth updating sop to include steps to take to include checking if pre-flights passed.

Reply:
Yes, if the installation fails for any reason from the beginning of the code we should have some mechanism for that to be flagged with SRE. Agree we should have SOP cases which cover pre-flight failures as well.